lambda/fulfillment/lib/middleware/2_preprocess.js

```diff
@@ -76,14 +76,15 @@ module.exports = async function preprocess(req, res) {
     let prehook = _.get(req, '_settings.LAMBDA_PREPROCESS_HOOK', undefined) || process.env.LAMBDA_PREPROCESS
     _.set(req, "_fulfillment.step", "preprocess")
     if (prehook) {
+        let arn = util.getLambdaArn(prehook)
         let result = await util.invokeLambda({
```

Comment:
For calling the prehook or posthook lambda if specified in the configuration, does the lambda name need to be prefixed with qna-* or QNA-* as per the InvokePolicy (defined in fulfilment.js) and used in FulfillmentLambdaRole?
```
...
          "Resource": [
            "arn:aws:lambda:*:*:function:qna-*",
            "arn:aws:lambda:*:*:function:QNA-*",
...
```

If so, some questions:
1) Should we note that requirement in docs/settings.md ?
2) Should we validate the lambda name prefix (in the config and/or in the 2_preprocess and 4_hooks)?

Reply:
Done